server/container/guice/configuration/src/main/java/org/apache/james/utils/DelegatedPropertiesConfiguration.java

```diff
@@ -372,6 +377,7 @@ public void unlock(LockMode mode) {
 
     private Stream<String> splitAndStripDoubleQuotes(String value) {
         return Optional.ofNullable(value)
+            .filter(s -> !StringUtils.isAllBlank(s))
```

Comment:
shouldn't filtering come after
```
.map(String::trim) ?
```
instead ? to avoid leaving empty values such as `" "` flow through and be converted to `""` ?

Reply:
Right!

Reply:
but then the first one on line 380 becomes useless :p